packages/database-types/package.json

```diff
@@ -11,7 +11,6 @@
     "index.d.ts"
   ],
   "peerDependencies": {
```

Comment:
Should this become a regular dependency instead?

Reply:
Do you mean making `@firebase/app` a regular dependency instead of removing it? No, that would install `@firebase/app` in Admin SDK causing version conflict potentially.

Or do you just mean renaming it to `dependencies` ?

Reply:
Not if we database-types a dev dependency of database.

Reply:
hmm, good catch. I think we probably should make app-types a dependency of database-types, or tsc will complain app-types is not found.

Reply:
hmm, good catch. I think we probably should do it, or tsc may complain app-types is not found.

Reply:
Or we can keep it as a peerDependency here and make app-types a dependency in admin sdk. WDTY?

Reply:
Made `app-types` a regular dependency, so we don't entirely lose the dependency.